### PR TITLE
UI bugfix.

### PR DESCRIPTION
**Pull request by zerai**
hi,
it should fix "[Copy all badges](https://github.com/PUGX/badge-poser/issues/445)" issue
sorry i cant provide an automated test.

### CODE REVIEW

File: assets/js/elements-utils.js [hunk 1]
```diff
@@ -10,3 +10,12 @@ export function removeChildren(node){
         node.removeChild( node.firstChild );
     }
 }
+
+export function makeCopyAllBadgesButtonElement(badgesAsMarkdown) {
```
Comment (JellyBellyDev) [inline assets/js/elements-utils.js:14]: this method doesn't seem like a utils to me, but it contains domain logic and in my opinion it should stay inside the file app.js! Such as the method `renderBadgeContainer()`. no?

File: assets/js/elements-utils.js [hunk 2]
```diff
@@ -10,3 +10,12 @@ export function removeChildren(node){
...
+    let div = document.createElement("div");
```
Comment (JellyBellyDev) [inline assets/js/elements-utils.js:15]: you couldn't use the `makeElement()` utils?

File: assets/js/elements-utils.js [hunk 3]
```diff
@@ -10,3 +10,12 @@ export function removeChildren(node){
...
+    div.innerHTML = `<button className="big" id="all-badges" data-clipboard-text="${badgesAsMarkdown.trim()}">
+        Copy All Badges
+    </button>`;
```
Comment (JellyBellyDev) [inline assets/js/elements-utils.js:18]: you couldn't use a specific function that return html, like as `renderBadge()`?

File: assets/js/app.js [hunk 4]
```diff
@@ -51,6 +51,15 @@ function renderBadges(badges) {
     badges.badges.forEach((badge) => {
         badgesContainer.appendChild(makeElement(renderBadgeContainer(badge)));
     });
+
+    let copyAllBadges = document.querySelector(".copy");
+    let badgesAsMarkdown = "";
+
+    removeChildren(copyAllBadges);
```
Comment (JellyBellyDev) [inline assets/js/app.js:58]: how about moving it after populating `badgesAsMarkdown `?

**Review by JellyBellyDev — CHANGES_REQUESTED** [review]
Thanks a lot @zerai for your contributing!!! 💪

File: assets/js/elements-utils.js (hunk 1 above)
Comment (zerai) [reply assets/js/elements-utils.js:14]: :+1:

File: assets/js/elements-utils.js (hunk 2 above)
Comment (zerai) [reply assets/js/elements-utils.js:15]: :+1:

File: assets/js/elements-utils.js (hunk 3 above)
Comment (zerai) [reply assets/js/elements-utils.js:18]: :+1:

File: assets/js/app.js (hunk 4 above)
Comment (zerai) [reply assets/js/app.js:58]: I will remove badgesAsMarkdown variable and the extra loop there is a better/clean way.

For consistency reason, the new code should follow same flow as the previous code blocks. IMO

- select the element
- remove the element
- populate the dom with a new element

WDYT?

Comment (JellyBellyDev) [reply assets/js/app.js:61]: Amazing!!! 🚀
Thanks a lot!!!

**Review by JellyBellyDev — APPROVED** [review]
LGTM

**Pull request merged**